1. Desktop app calls Web API/AppCreationScripts/Configure.ps1

```diff
@@ -248,15 +248,16 @@ Function ConfigureApplications
    ReplaceSetting -configFilePath $configFile -key "todo:TodoListScope" -newValue ("api://"+$serviceAadApplication.AppId+"/user_impersonation")
    ReplaceSetting -configFilePath $configFile -key "todo:TodoListBaseAddress" -newValue $serviceAadApplication.HomePage
    Write-Host ""
+   Write-Host -ForegroundColor Green "------------------------------------------------------------------------------------------------" 
    Write-Host "IMPORTANT: Please follow the instructions below to complete a few manual step(s) in the Azure portal":
    Write-Host "- For 'service'"
    Write-Host "  - Navigate to '$servicePortalUrl'"
-   Write-Host "  - Navigate to the Manifest page and change 'signInAudience' to 'AzureADandPersonalMicrosoftAccount'."
-   Write-Host "  - Still in the Manifest page, change 'accessTokenAcceptedVersion' to 2 "
+   Write-Host "  - Navigate to the Manifest page and change 'signInAudience' to 'AzureADandPersonalMicrosoftAccount'." -ForegroundColor Red 
+   Write-Host "  - Still in the Manifest page, change 'accessTokenAcceptedVersion' to 2 " -ForegroundColor Red 
```

Comment:
did you change the generator, @kalyankrishna1  ?

Reply:
@kalyankrishna1 : will you merge?

Reply:
Yes I did